src/variety/cbblock.js

```diff
@@ -38,6 +74,17 @@ Border:{
 	isGround : function(){ return (this.ques>0);}
 },
 
+"Border@dbchoco":{
+	isGround : function(){
```

Comment:
do you maybe have an idea for a clearer name than `isGround`? it makes less sense here than for cbblock

Reply:
`isGround` has been removed for dbchoco.